legal-api/src/legal_api/utils/datetime.py

```diff
@@ -14,8 +14,7 @@
 """Date time utilities."""
 # from datetime import datetime, timezone
 import time as _time
-from datetime import date, datetime as _datetime, timezone, \
-    timedelta  # pylint: disable=unused-import # noqa: F401, I001, I005
+from datetime import date, datetime as _datetime, timezone  # pylint: disable=unused-import # noqa: F401, I001, I005
```

Comment:
Why not delete the unused import instead disabling the lint issue ?

Reply:
This import is needed. Some other classes import from here.